tools/run_tests/run_xds_tests.py

```diff
@@ -56,17 +56,27 @@
     'secondary_locality_gets_requests_on_primary_failure',
     'traffic_splitting',
 ]
+# Valid test cases, but not in all. So the tests can only run manually, and
+# aren't enabled automatically for all languages. Move them into _TEST_CASES
+# when support is ready in all languages.
+_ALTERNATIVE_TEST_CASES = ['path_matching', 'header_matching']
```

Comment:
Optional, Nit: alternative isn't great for a name here. I think `_NEW_TEST_CASES` or `_ADDITIONAL_TEST_CASES` would, well, not be that much better...`_NOT_UNIVERSALLY_IMPLEMENTED_TEST_CASES` would be more precise but a little verbose.

Reply:
Changed to `_ADDITIONAL_TEST_CASES`